### PR TITLE
pythonPackages.junitparser: init at 1.4.1

### PR DESCRIPTION
**Pull request by multun**
###### Motivation for this change

Packaging closed source software for NixOS!

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/development/python-modules/junitparser/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    owner = "gastlygem";
+    repo = pname;
+    rev = version;
+    sha256 = "16xwayr0rbp7xdg7bzmyf8s7al0dhkbmkcnil66ax7r8bznp5lmp";
+  };
+
+  propagatedBuildInputs = [ future ];
+
+  checkPhase = ''
+    python test.py
```
Comment (jonringer) [inline pkgs/development/python-modules/junitparser/default.nix]: to support other interpreters such as pypy
```suggestion
    ${python.interpreter} test.py
```

Comment (multun) [reply pkgs/development/python-modules/junitparser/default.nix]: @jonringer fixed, thanks!

**Review by jonringer — APPROVED** [review]
LGTM
executable shows usage

```
[5 built, 4 copied (8.9 MiB), 1.8 MiB DL]
https://github.com/NixOS/nixpkgs/pull/76658
3 package were built:
python27Packages.junitparser python37Packages.junitparser python38Packages.junitparser
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build python27Packages.junitparser python37Packages.junitparser python38Packages.junitparser

**Pull request merged**